controllers/selfnoderemediation_controller.go

```diff
@@ -600,11 +628,20 @@ func (r *SelfNodeRemediationReconciler) updateSnrStatus(ctx context.Context, snr
 	return nil
 }
 
-func (r *SelfNodeRemediationReconciler) updateTimeAssumedRebooted(node *v1.Node, snr *v1alpha1.SelfNodeRemediation) {
-	r.logger.Info("updating snr with node backup and updating time to assume node has been rebooted", "node name", node.Name)
-	//we assume the unhealthy node will be rebooted by maxTimeNodeHasRebooted
-	maxTimeNodeHasRebooted := metav1.NewTime(metav1.Now().Add(r.SafeTimeCalculator.GetTimeToAssumeNodeRebooted()))
-	snr.Status.TimeAssumedRebooted = &maxTimeNodeHasRebooted
+func (r *SelfNodeRemediationReconciler) setTimeAssumedRebooted(ctx context.Context, node *v1.Node, snr *v1alpha1.SelfNodeRemediation) {
+	if snr.Status.TimeAssumedRebooted != nil {
+		// timeAssumedRebooted already set, nothing to do
+		return
+	}
+	// get reboot duration
+	rebootDuration, err := r.RebootDurationCalculator.GetRebootDuration(r.Client, ctx, node, r.MyNamespace)
```

Comment:
I don't think it makes sense to ignore the error.

I suggest to either:
 - modify GetRebootDuration not to return an error (it can return the Fallback time in case it can't calculate the time) 
 - or address the returned error.

Reply:
yes, in the context of returning an error when the config isn't set yet in the calculator, I already modified this to log and return the error, in order to always retry. There is a good chance that every possible error is temporary only. That way we also don't need the fallback.